packages/filesystem/src/browser/filesystem-preferences.ts

```diff
@@ -43,7 +43,13 @@ export const filesystemPreferenceSchema: PreferenceSchema = {
             'default': { '**/.git': true, '**/.svn': true, '**/.hg': true, '**/CVS': true, '**/.DS_Store': true },
             'description': 'Configure glob patterns for excluding files and folders.',
             'scope': 'resource'
-        }
+        },
+        'files.associations': {
```

Comment:
Should not it be used somewhere? I.e. and editor language should be changed according it? not sure why it was merged and an issue was closed

Reply:
it's used by inteliphense feature there https://github.com/bmewburn/vscode-intelephense/search?q=files.associations&unscoped_q=files.associations

AFAIK you said it's important that master contains fixes for ppl that are blocked
https://github.com/theia-ide/theia/pull/4521#issuecomment-472007892

Reply:
It was finished thing, this code adds user feature which accessible but does not work.

Reply:
it allows several VS Code extensions to work. https://github.com/search?q=getConfiguration%28%27files%27%29.get%28%27associations%27%29%3B&type=Code

Please, what are you calling "finished thing" ? something that is making Theia accepting all VS Code extensions in a single commit ?

Reply:
> it allows several VS Code extensions to work. https://github.com/search?q=getConfiguration%28%27files%27%29.get%28%27associations%27%29%3B&type=Code

i wonder how, even if extensions contribute it, it does not take any effect, it does not work

Reply:
well it allows the extension to be deployed ?

Reply:
Could not it be then just to be mocked in plugin-ext for now, without introducing user visible preferences?

Reply:
> Please, what are you calling "finished thing" ? something that is making Theia accepting all VS Code extensions in a single commit ?

For sure not, but one always has to consider how it affects existing users who don't even use VS Code extensions.

Reply:
@tolusha could it be introduced only in plugin-ext ?

Reply:
ok, I will introduce a new PR.